packages/badge/README.md

```diff
@@ -159,3 +215,40 @@ When you'd like to have the `<sp-badge>` display as if "fixed" to the edge of a
     </sp-badge>
 </div>
 ```
+
+### Behaviors
+
+Badges are not interactive by default.
+
+When a badge's label is too long for the available horizontal space, it wraps to form another line. Text wrapping can be enforced when a `max-inline-size` is applied to the badge. If there is no room for a second line of text, the badge should truncate and include a tooltip to expose the full text upon hover.
+
+```html demo
+<sp-badge style="max-inline-size: 350px;">
+    Wikipedia is the best thing ever. Anyone in the world can write anything
+    they want about any subject so you know you are getting the best possible
+    information.
+</sp-badge>
+```
+
+### Accessibility
+
+#### Always include a label
```

Comment:
This is great, but can you add guidance on ARIA attributes. For example, using aria-label for icon-only badges and ensuring tooltips are accessible. Also can you clarify keyboard navigation, especially if tooltips are triggered on focus as well as hover.